rotkehlchen/data/all_assets.json

```diff
@@ -7377,6 +7377,16 @@
         "symbol": "MCI",
         "type": "ethereum token"
     },
+    "MUST": {
+        "coingecko": "Must",
+        "cryptocompare": "",
+        "ethereum_address": "0x9c78ee466d6cb57a4d01fd887d2b5dfb2d46288f",
+        "ethereum_token_decimals": 18,
+        "name": "MUST (Cometh)",
+        "started": 1610699161,
```

Comment:
Where is this timestamp coming from? Checking the [transaction](https://etherscan.io/tx/0x394e686870eccb31abf47f91d552e6b2013094d85439030252dc0c1f42baaa65) the contract was deployed the timestamp at time of deployment should be 1610742361 right?

Reply:
You're right? I did for 8AM instead of 8PM 🤦